packages/cli/src/server/web/mod.rs

```diff
@@ -369,8 +369,8 @@ async fn start_server(
     #[cfg(feature = "plugin")]
     PluginManager::on_serve_start(_config)?;
 
-    // Parse address
-    let addr = format!("0.0.0.0:{}", port).parse().unwrap();
+    // Bind the server to `[::]` and it will LISTEN for both IPv4 and IPv6. (required IPv6 dual stack)
+    let addr = format!("[::]:{}", port).parse().unwrap();
```

Comment:
Wrangler uses `localhost` when proxying static files to `dx serve`. `localhost` resolves to an IPv6 Lookback (`[::1]`) on Windows, but dx serve only LISTENS to IPv4, so the proxy fails.
So I would like to change it so that dx serve LISTENs to IPv6 dual stack.